code/modules/jobs/job_types/bos.dm

```diff
@@ -227,39 +203,30 @@ Scribe
 /*
 Initiate Knight
 */
-/datum/job/f13initiateknight
+/datum/job/bos/f13initiateknight
 	title = "Initiate Knight"
 	flag = F13INITIATEKNIGHT
-	department_flag = BOS
 	faction = "BOS"
-	total_positions = 1
-	spawn_positions = 1
+	total_positions = 2
+	spawn_positions = 2
 	description = "You answer directly to the Knights, as an inexperienced member of the brotherhood your sole purpose is to train in order to become a fully-fledged Knight in the future. You should take care in not straying from the bunker without an escort. The Brotherhoods presence in the Mojave is unknown to most outsiders, so you should take care in making sure that wastelanders do not recognize your identity, or find the location of your bunker."
 	forbids = "The Brotherhood of Steel Forbids: Unethical human experimentation. Violence beyond what is needed to accomplish Brotherhood goals."
 	enforces = "The Brotherhood of Steel Expects: Obeying the Chain That - Binds your direct superior. Collection and safeguarding of technology from the wasteland. Experimentation and research."
 	supervisors = "the knights"
 	selection_color = "#95a5a6"
 
 
-	outfit = /datum/outfit/job/f13initiateknight
+	outfit = /datum/outfit/job/bos/f13initiateknight
 
 	access = list(ACCESS_BOS)
 	minimal_access = list(ACCESS_BOS)
 
-/datum/outfit/job/f13initiateknight
+/datum/outfit/job/bos/f13initiateknight
 	name = "Initiate Knight"
-	jobtype = /datum/job/f13initiateknight
-	backpack = /obj/item/storage/backpack/explorer
-	ears = 			/obj/item/radio/headset/headset_bos
-	uniform =		/obj/item/clothing/under/f13/recon
-	shoes = 		/obj/item/clothing/shoes/combat/swat
-	gloves = 		/obj/item/clothing/gloves/combat
+	jobtype = /datum/job/bos/f13initiateknight
 	suit = 			/obj/item/clothing/suit/armor/f13/combat/brotherhood
 	head = 			/obj/item/clothing/head/helmet/f13/combat/brotherhood
 	suit_store =	/obj/item/gun/ballistic/automatic/marksman/servicerifle
```

Comment:
give them an aer9 instead of a service rifle, more thematically appropriate

Reply:
Done.